image/gimp_brush.ksy

```diff
@@ -0,0 +1,38 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
+  endian: be
+doc-ref: https://gitlab.gnome.org/GNOME/gimp/-/raw/441631322be109da6489b2aad670bdba916315c0/devel-docs/gbr.txt
+seq:
+  - id: len_header
+    type: u4
+  - id: header
+    type: header
+    size: len_header
+types:
+  header:
+    seq:
+      - id: version
+        type: u4
+      - id: width
+        type: u4
+      - id: height
+        type: u4
+      - id: bytes_per_pixel
+        type: u4
+      - id: magic
+        contents: GIMP
+      - id: spacing
+        type: u4
+      - id: brush_name
+        type: strz
+        size-eos: true
+        encoding: UTF-8
+instances:
+  body_size:
+    value: header.width * header.height * header.color_depth
```

Comment:
You need to rename `color_depth` here as well (https://github.com/kaitai-io/kaitai_struct_formats/pull/427/commits/56f7c1c4d877f49547f2b7dcb0b3305d0860196e):

```suggestion
    value: header.width * header.height * header.bytes_per_pixel
```